.github/CODEOWNERS

```diff
@@ -24,8 +24,11 @@
 /teams/team-repo-admins.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
 /people/Mark-Simulacrum.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
 /people/Nadrieril.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
+/people/Noratrieb.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
+/people/apiraino.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
 /people/jackh726.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
 /people/jdno.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
+/people/jieyouxu.toml @Mark-Simulacrum @pietroalbini @jdno @marcoieni
```

Comment:
these people have write access to the repo, so it was a bug that they weren't present in the codeowners file